multires-lyrics-search/tests/test_multireslyrics.py

```diff
@@ -91,13 +93,18 @@ def test_query(tmpdir, queries_and_expected_replies):
             query_chunk_results = []
             for chunk in chunks:
                 chunk_result = {'chunk': chunk['text'], 'chunk_matches': []}
-                for match in chunk['matches']:
+                chunk_matches = chunk['matches']
+                # make sure to sort in asc. order, by score
+                chunk_matches = sorted(chunk_matches, key=lambda x: x['score']['value'], reverse=False)
+                for match in chunk_matches:
                     chunk_result['chunk_matches'].append(match['text'])
                 query_chunk_results.append(chunk_result)
             assert query_chunk_results == exp_result["chunk-level"]
 
             # match-level comparison
             matches = output['search']['docs'][0]['matches']
+            # make sure to sort in asc. order, by score
+            matches = sorted(matches, key=lambda x: x['score']['value'], reverse=False)
             match_result = []
             for match in matches:
                 match_text = match['text']
```

Comment:
why do not we have an assert at `doc level` of matches anymore?

Reply:
You mean this https://github.com/jina-ai/examples/blob/a38608b685e426fbb412462147d093a0bd7509b2/multires-lyrics-search/tests/test_multireslyrics.py#L112 ?

Reply:
Maybe it should be renamed to `"doc-level"`

Reply:
Ah no my bad, yes the diff only showed the changes and got me confused.